src/perl/vdotest/VDOTest/Vdorecover.pm

```diff
@@ -213,11 +214,13 @@ sub testFullFilesystem {
                   );
   $dir2->pollUntilDone();
 
-  $self->syncDeviceIgnoringErrors();
-
   # The error syncing matches either No space left on device or Input/output
   # error, but distinguishing them is hard.
+  $self->syncDeviceIgnoringErrors();
+  $fs->unmount();
+  $device->runOnHost("sudo dmesg -c > /dev/null");
 
+  # Run the vdorecover utility
```

Comment:
This is just defining the steps to run the utility, not actually run it yet.  Would suggest altering the comment to something along the lines of `# Define the recovery utility execution process`.  Happy to see other suggestions.